src/main/java/org/reaktivity/nukleus/tls/internal/TlsNukleus.java

```diff
@@ -143,7 +143,7 @@ public CommandHandler commandHandler(
     @Override
     public TlsElektron supplyElektron()
     {
-        return new TlsElektron(config, this::findStore);
+        return new TlsElektron(config, routeId -> findStore(storesByRouteId.get(routeId)), this::findStore);
```

Comment:
On the client, we don't use storesByRouteId and we know the store name I thought of using `findStore` instead.